pypit/arparse.py

```diff
@@ -994,6 +994,20 @@ def reduce_calibrate_wavelength(self, v):
         v = key_allowed(v, allowed)
         self.update(v)
 
+    def reduce_detnum(self, v):
+        """ Reduce only the input detector of the array
+
+        Parameters
+        ----------
+        v : str
+          value of the keyword argument given by the name of this function
+        """
+        if v.lower() == 'none':
```

Comment:
I would encourage you to write a new function called `key_none_int`. There is a similar example in `arparse.py` already, called `key_none_list`.